G/GlibNetworking/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GlibNetworking"
+version = v"2.74.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/glib-networking/2.74/glib-networking-$(version).tar.xz", "1f185aaef094123f8e25d8fa55661b3fd71020163a0174adb35a37685cda613b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/glib-networking-*
+install_license COPYING
+
+mkdir build-glib && cd build-glib
+meson --cross-file=${MESON_TARGET_TOOLCHAIN} --buildtype=release ..
+
+# Meson beautifully forces thin archives, without checking whether the dynamic linker
+# actually supports them: <https://github.com/mesonbuild/meson/issues/10823>.  Let's remove
+# the (deprecated...) `T` option to `ar`
+sed -i.bak 's/csrDT/csrD/' build.ninja
+
+ninja -j${nproc}
+
+# We cannot use `ninja install` here since it would run a script
+# which cannot work in cross-compile environments.
+mkdir -p ${libdir}/gio/modules
+cp proxy/environment/libgioenvironmentproxy.so ${libdir}/gio/modules/
+cp tls/gnutls/libgiognutls.so ${libdir}/gio/modules/
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; exclude=Sys.iswindows)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libgioenvironmentproxy", :libgioenvironmentproxy, "lib/gio/modules"),
```

Comment:
[Plugins](https://eli.thegreenplace.net/2012/08/24/plugins-in-c) aren't `LibraryProduct`s.  At most, you can have it as a `FileProduct`.